src/common.ts

```diff
@@ -37,3 +37,11 @@ export const formatBitcoinCashAddress = (address: string) => {
   if (address.startsWith('bitcoincash:')) address = address.slice(12)
   return address
 }
+
+export const isValidSolanaAddress = (address: string): boolean => {
+  return true
```

Comment:
let's add some checks here even if they are simply checking the format and length
Same with tx validation